src/rapidata/rapidata_client/assets/media_asset.py

```diff
@@ -21,68 +22,76 @@ class MediaAsset(BaseAsset):
         FileNotFoundError: If the provided file path does not exist.
     """
 
+    ALLOWED_TYPES = [
+        'image/', 
+        'audio/mp3',      # MP3
+        'video/mp4',       # MP4
+    ]
+
     def __init__(self, path: str):
         """
         Initialize a MediaAsset instance.
 
         Args:
-            path (str): The file system path to the media asset or a link to an image.
+            path (str): The file system path to the media asset or a URL.
 
         Raises:
             FileNotFoundError: If the provided file path does not exist.
+            ValueError: If media type is unsupported or duration exceeds 25 seconds.
         """
         if not isinstance(path, str):
-            raise ValueError("Media must be a string, either a local file path or an image URL")
+            raise ValueError("Media must be a string, either a local file path or a URL")
 
         if re.match(r'^https?://', path):
-            self.path = MediaAsset.get_image_bytes(path)
+            self.path = self._get_media_bytes(path)
             self.name = path.split('/')[-1]
-            if not self.name.endswith(('.jpg', '.jpeg', '.png', '.gif')):
-                self.name += '.jpg'
+            if not self.name.endswith(('.jpg', '.jpeg', '.png', '.gif', '.mp3', '.mp4')):
+                raise ValueError("Supported file types for custom names: jpg, jpeg, png, gif, mp3, mp4")
```

Comment:
we do not support .webp? I am not really sure where we support it and where not tbh

Reply:
I think webp should work fine

Reply:
good catch! thanks!